lib/utils/azure_queues.ts

```diff
@@ -14,16 +20,14 @@ import { QueueService } from "azure-storage";
 //    before putting the message into the poison queue
 // if (MAX_RETRIES > maxDequeueCount - 1) the system will retry even after maxDequeueCount
 //    is reached and duplicate messages will be put into the poison queue
-export const RETRIES: ReadonlyArray<number> = [
-  10,
-  60,
-  600,
-  3600,
-  21600,
-  86400,
-  172800,
-  345600
-];
+export const RETRIES: ReadonlyArray<number> = unfoldr((retries: number): Option<
```

Comment:
you should make this a function now `(retry: number) => number`

Reply:
I need to know in advance how many retries can be scheduled in the the specified time to live (RETRIES.length) to check that maxDequeCount is setup correctly in hosts.json

Reply:
`max_retries < log_2(TIME_TO_LIVE / MIN_BACKOFF)`

Reply:
`max_retries = [ log_2(7 * 24 * 3600 * 1000 / 285) ] = 21`

perhaps there's a 1 off difference from your implementation, anyway it's simple algebra, you don't have to precalculate all values

Reply:
of course. it was just to don't have to write the logic (pow) and the reverse logic (log) and then keep them in sync (I'll turn it to a function anyway)